website/content/en/docs/concepts/scheduling.md

```diff
@@ -200,6 +208,10 @@ When setting rules, the following Node affinity types define how hard or soft ea
 * **requiredDuringSchedulingIgnoredDuringExecution**: This is a hard rule that must be met.
 * **preferredDuringSchedulingIgnoredDuringExecution**: This is a preference, but the pod can run on a node where it is not guaranteed.
 
+{{% alert title="Note" color="primary" %}}
+Karpenter uses preferred as if it was required when determining if a pod can schedule on a node (absent topology requirements), or when determining if a pod can be shifted to a new node. When preferred affinities can't be satisfied by any possible nodeclaim configuration then Karpenter will relax the preferences and only consider required affinities. In practice, this behavior can result in more nodes being created than expected, as Karpenter will prefer to create new nodes to satisfy preferences.
```

Comment:
```suggestion
Karpenter treats preferred affinities as if they are required when determining if a pod can schedule on a node (absent topology requirements), or when determining if a pod can be shifted to a new node. When preferred affinities can't be satisfied by any possible nodeclaim configuration then Karpenter will relax the preferences and only consider required affinities. In practice, this behavior can result in more nodes being created than expected, as Karpenter will prefer to create new nodes to satisfy preferences.
```

Does it make sense to keep this statement shortened and then link up to the preferences section in this doc? That way -- you have a single source of truth for talking about preferences in the scheduling doc and don't have to keep referencing it across the different scheduling semantics that we cover

Reply:
Great idea. The docs in the Preferences section has the content, I've updated the alerts to reduce their verbiage and to reference back to it.